pkg/windows/windows.go

```diff
@@ -137,8 +137,8 @@ const (
 	ManagedTag = "OpenShift managed"
 	// containersFeatureName is the name of the Windows feature that is required to be enabled on the Windows instance.
 	containersFeatureName = "Containers"
-	// wicdKubeconfigPath is the path of the kubeconfig used by WICD
-	wicdKubeconfigPath = K8sDir + "\\wicd-kubeconfig"
+	// WicdKubeconfigPath is the path of the kubeconfig used by WICD
+	WicdKubeconfigPath = K8sDir + "\\wicd-kubeconfig"
```

Comment:
`WICDKubeconfigPath` as it is an acronym

Reply:
Addressed, will push after the test run finishes.

https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_windows-machine-config-operator/2636/pull-ci-openshift-windows-machine-config-operator-master-azure-e2e-operator/1882145294536151040